_scicompannounce/2022-11-12-memory-scheduling.md

```diff
@@ -0,0 +1,53 @@
+---
+title: Memory Scheduling on Gizmo
+---
+
+Starting Xxxxxx Xth, 20xx we will begin constraining the memory your job uses. This will require that you add a memory request to your job as any job that goes over its memory request will be killed.
+
+## Background
+
+To date we have not controlled the amount of memory your jobs use, instead relying on the number of cores (or CPUs) as a loose proxy for the amount of memory your job will need.
+
+As memory available to all jobs on a node is otherwise unconstrained, we frequently encounter situations where a job uses more memory than availble or drives memory use to a point where the node fails.  This impacts everyone allocated to that node, causing all jobs on the node to fail.
+
+## The Change
+
+We will be updating the configuration of Slurm to constrain memory available to a job.  Jobs that attempt to allocate more memory will fail.  Jobs that do not explictly request memory will be given a default request of 10MB per core requested.  The maximum memory per core is 20GB.
```

Comment:
Not sure if this is closed forever, but did you really mean "10MB per core requested" and not 10GB?